inc/class-tool.php

```diff
@@ -660,4 +671,227 @@ function ( $var ) {
 		return true;
 	}
 
+	/**
+	 * Process incoming request, check authenticity of the LTI launch request
+	 *
+	 * @since 1.4.0
+	 *
+	 * @param array $params
+	 */
+	public function processRequest( $params ) {
+		$this->setParams( $params );
+		$this->setParameterConstraint(
+			'oauth_consumer_key', true, 50, [
```

Comment:
`50` is repeated 3 times in this method, can you convert it to a constant?

Reply:
excellent, thank you!